src/utils/openSea.ts

```diff
@@ -0,0 +1,36 @@
+import axios from "axios"
+import queryString from "query-string"
+
+import env from "@/.env.json"
+
+const OPENSEA_API_BASE_URL =
+  env.env === "development"
+    ? "https://rinkeby-api.opensea.io/api/v1"
+    : "https://api.opensea.io/api/v1"
+
+type OpenSeaAsset = {
+  token_id: string
```

Comment:
there is also a `permalink` might be better to define `toOpenSeaNFTUrl` below?

            "permalink": "https://testnets.opensea.io/assets/0xae89d81ab5c668661200fa9c6ed45fe1707f7097/125",

Reply:
cool, will resolve it